frontend/src/scenes/sceneLogic.ts

```diff
@@ -154,6 +159,7 @@ export const routes: Record<string, Scene> = {
     '/action/:id': Scene.Action,
     '/action': Scene.Action,
     '/insights': Scene.Insights,
+    '/i/:id': Scene.InsightRouter,
```

Comment:
What's the reasoning behind using `i` rather than `insight`. Seems like it would be more consistent

Reply:
Just having a shorter URL, like Youtube's `https://youtu.be/dQw4w9WgXcQ`. When you open these links you get redirected to the actual video (or insight in our case) page. POV?